src/timedisc/timedisc.jl

```diff
@@ -23,6 +23,7 @@ class 2N of Williamson.
       end
     end
   end
+  apply_positivity_preserving_limiter!(solver)
```

Comment:
If we really want to have some guarantee, we would need to

- use an SSP method and
- `apply_positivity_preserving_limiter!(solver)` after every explicit Euler sub-step.

But it's okay for me to leave it as it is right now.

Reply:
I actually call the PP limiter in the RHS, before the DG operator is called. So, it is used in every explicit Euler sub-step